pylxd/__init__.py

```diff
@@ -14,6 +14,6 @@
 
 import pbr.version
 
-__version__ = pbr.version.VersionInfo('pylxd').version_string()
+__version__ = pbr.version.VersionInfo("pylxd").version_string()
 
-from pylxd.client import Client,EventType  # NOQA
+from pylxd.client import Client, EventType  # NOQA
```

Comment:
unrelated, but we could drop these NOQA and declare those imports in `__all__` (as it's the public API)